pkg/blobstore/circular/circular_blob_access.go

```diff
@@ -59,37 +61,49 @@ func NewCircularBlobAccess(offsetStore OffsetStore, dataStore DataStore, stateSt
 }
 
 func (ba *circularBlobAccess) Get(ctx context.Context, digest *util.Digest) (int64, io.ReadCloser, error) {
+	ctx, span := trace.StartSpan(ctx, "circularBlobAccess.Get")
+	defer span.End()
 	ba.lock.Lock()
+	span.Annotate(nil, "Lock obtained")
 	cursors := ba.stateStore.GetCursors()
 	offset, length, ok, err := ba.offsetStore.Get(digest, cursors)
+	span.Annotate([]trace.Attribute{trace.Int64Attribute("offset", int64(offset)), trace.Int64Attribute("length", length), trace.BoolAttribute("object_found", ok)}, "offsetStore.Get completed")
 	ba.lock.Unlock()
 	if err != nil {
 		return 0, nil, err
 	} else if ok {
+		span.Annotate(nil, "Obtaining body ReadCloser")
 		return length, ba.dataStore.Get(offset, length), nil
 	}
 	return 0, nil, status.Errorf(codes.NotFound, "Blob not found")
 }
 
 func (ba *circularBlobAccess) Put(ctx context.Context, digest *util.Digest, sizeBytes int64, r io.ReadCloser) error {
 	defer r.Close()
+	ctx, span := trace.StartSpan(ctx, "circularBlobAccess.Put")
+	defer span.End()
 
 	// Allocate space in the data store.
 	ba.lock.Lock()
+	span.Annotatef(nil, "Lock obtained, allocating %d bytes", sizeBytes)
 	offset, err := ba.stateStore.Allocate(sizeBytes)
 	ba.lock.Unlock()
 	if err != nil {
 		return err
 	}
+	span.Annotatef(nil, "Store allocated, offset %d", offset)
 
 	// Write the data to storage.
 	if err := ba.dataStore.Put(r, offset); err != nil {
 		return err
 	}
 
+	span.Annotate(nil, "Obtaining lock")
 	ba.lock.Lock()
+	span.Annotate(nil, "Lock obtained, calling GetCursors")
```

Comment:
Nit: stay consistent with [previous annotation](https://github.com/buildbarn/bb-storage/pull/10/files#diff-b7987a20374b1951b942f7bacdd9f27aR67).

Reply:
Could you please be more specific?

Reply:
Taking a look at [this annotation](https://github.com/buildbarn/bb-storage/pull/10/files#diff-b7987a20374b1951b942f7bacdd9f27aR67), and [this one](https://github.com/buildbarn/bb-storage/pull/10/files#diff-b7987a20374b1951b942f7bacdd9f27aR103), they are actually on the same logic (i.e. getting a lock and calling getCursors). It would be nice if the annotations were the same, given that the logic stays identical across those traces.